src/sentry/tasks/app_store_connect.py

```diff
@@ -42,6 +43,8 @@ def inner_dsym_download(project_id: int, config_id: str) -> None:
     config = appconnect.AppStoreConnectConfig.from_project_config(project, config_id)
     client = appconnect.AppConnectClient.from_config(config)
 
+    build_refresh_dates = project.get_option(appconnect.SYMBOL_SOURCE_REFRESH_PROP_NAME, default={})
```

Comment:
Might as well move this to right before line 59, it reduces the time we have this loaded before stored again